app/event_ethdenver2019/views.py

```diff
@@ -80,7 +80,12 @@ def ethdenver2019(request):
         "page2": request.GET and recv_addr != 'invalid',
         "page3": all_kudos_collected
     }
+
     page_ctx["page1"] = not page_ctx['page2']
+    if page_ctx["page1"] == True:
```

Comment:
shouldnt this be triggered by something dynamic, like the system knowing that hte user collected all kduos?

Reply:
This is just to avoid that the page3 (== the 'hooray') is displayed on the "Get started" page.

Reply:
@kuhnchris does the system recognize when all of the kudos have been captured, and show `page3` ? if so, where can i see that code?

Reply:
@kuhnchris does the system recognize when all of the kudos have been captured, and show `page3` ? if so, where can i see that code?

Reply:
Assuming all Kudos are collected...
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L56
If not then revert:
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L70
And then page3 (which isn't a page anymore but the 'popover') is set to that value:
https://github.com/gitcoinco/web/blob/c2e00abbb11afa0876918902e0ce9ba638c4bdb8/app/event_ethdenver2019/views.py#L78-L82